### PR TITLE
Backend for history-service

### PR DESCRIPTION
**Pull request by ongweisheng**
- Created schema for history-service
- API to create history
- API to get history
- Tests will be subsequently added in future PRs

### CODE REVIEW

File: history-service/models/history.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        type: String,
+        required: true,
+    },
+    question: {
+        type: Object,
+        required: true,
+    }
+})
+
+export default mongoose .model("History", historySchema)
```
Comment (TheSpaceCuber) [inline history-service/models/history.js:22]: spacing for `mongoose .model`

File: history-service/models/history.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose"
+
+const historySchema = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true,
+    },
+    matchUsername: {
```
Comment (TheSpaceCuber) [inline history-service/models/history.js:8]: what does matchUsername refer to? the other user?

File: history-service/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import express from 'express'
+import cors from 'cors'
+import mongoose from 'mongoose'
+import 'dotenv/config'
+import historyRoutes from "./routes/historyRoutes.js"
+
+let uri;
+
+if (process.env.ENV == "PROD") {
```
Comment (TheSpaceCuber) [inline history-service/index.js:9]: To discuss on standardization whether to use `ENV` or `NODE_ENV` next week. If we use `NODE_ENV` then the two possible values would be `development` and `production` by convention.

https://stackoverflow.com/questions/16978256/what-is-node-env-and-how-to-use-it-in-express

File: history-service/controllers/historyController.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import History from "../models/history.js"
+
+export const createHistory = async (req, res) => {
+    try {
+        const history = await History.create(req.body)
+        return res.status(200).json({ message: `interview history save for ${history.username} ` })
```
Comment (TheSpaceCuber) [inline history-service/controllers/historyController.js:6]: saved instead of save?

**Review by TheSpaceCuber — COMMENTED** [review]
Just some tweaks and comments. So for each match it will save 2 history documents? 1 for each user.

File: history-service/models/history.js (hunk 2 above)
Comment (ongweisheng) [reply history-service/models/history.js:8]: yup, refers to other user

**Review by TheSpaceCuber — APPROVED** [review]
LGTM

**Pull request merged**